Projects/Foundations/Schemas/Docs-Topics/I/Introduction/Introduction/introduction-006-nodes-and-structures-of-nodes.json

```diff
@@ -31,8 +31,8 @@
                 },
                 {
                     "language": "EL",
-                    "text": "Δομή Κόμβων",
-                    "updated": 1638972581810
+                    "text": "Structure of Nodes (Δομή Κόμβων)",
+                    "updated": 1641830315993
```

Comment:
You accidentally overwrite the Russian translation. Same problem in this File but absolutely fixable.